### PR TITLE
Feature/hck 4265 faker

### PR DESCRIPTION
**Pull request by ArzamastsevVladyslav**

### CODE REVIEW

File: reverse_engineering/helpers/fetchRequestHelper.js [hunk 1]
```diff
@@ -81,19 +83,74 @@ const destroyActiveContext = () => {
...
+	if (lineNumber < 1000) {
+		return lineNumber % 100 === 0;
+	}
+	return lineNumber % 1000 === 0;
+}
+
+
+const sendSampleBatches = (_, logger) => async (connectionInfo) => {
+	const { entitiesData } = connectionInfo;
+	const batchSize = 5000;
```
Comment (VitaliiBedletskyi) [inline reverse_engineering/helpers/fetchRequestHelper.js:112]: Maybe it is better to move `batchSize` to module level constant and use one value for batch insert and logging (line 103)

File: reverse_engineering/helpers/fetchRequestHelper.js [hunk 2]
```diff
@@ -81,19 +83,74 @@ const destroyActiveContext = () => {
...
+			},
+			logProgress: (lineIndex, amountOfLines) => {
+				if (!shouldLogStep(lineIndex)) {
+					return;
+				}
+				const progress = Number(lineIndex / amountOfLines);
+				const message = `Inserted ${lineIndex} lines out of ${amountOfLines}, progress ${progress.toFixed(2)}%`;
+				logger.log('info', { message }, 'SEND_SAMPLE_BATCHES');
+				logger.progress({ message });
+			},
```
Comment (VitaliiBedletskyi) [inline reverse_engineering/helpers/fetchRequestHelper.js:133]: Please move to separate function

File: reverse_engineering/helpers/fetchRequestHelper.js [hunk 3]
```diff
@@ -587,6 +646,7 @@ const getCommandExecutionResult = (query, options, commandOptions) => {
 					description: commandOptions,
 				};
 			}
+            // await sleep(500);
```
Comment (VitaliiBedletskyi) [inline reverse_engineering/helpers/fetchRequestHelper.js:649]: Get rid of commented code

File: reverse_engineering/helpers/fetchRequestHelper.js [hunk 4]
```diff
@@ -567,7 +626,7 @@ const getCommandExecutionResult = (query, options, commandOptions) => {
 				responseBody,
 			};
 		})
-		.then(body => {
+		.then(async body => {
```
Comment (VitaliiBedletskyi) [inline reverse_engineering/helpers/fetchRequestHelper.js:629]: Redundant async

**Review by VitaliiBedletskyi — CHANGES_REQUESTED** [review]

File: reverse_engineering/helpers/fetchRequestHelper.js (hunk 1 above)
Comment (ArzamastsevVladyslav) [reply reverse_engineering/helpers/fetchRequestHelper.js:124]: I made `batchSize` a module-level constant but the constant for logging has a completely different meaning and purpose, so it is not a good idea to reuse this value there

File: reverse_engineering/helpers/fetchRequestHelper.js [hunk 5]
```diff
@@ -553,6 +614,8 @@ const executeCommand = (connectionInfo, command, language = 'sql', logger) => {
 	});
 };
 
+// const sleep = (millis) => new Promise((res, rej) => setTimeout(() => {res(true)}, millis));
```
Comment (mtseluiko) [inline reverse_engineering/helpers/fetchRequestHelper.js:617]: remove unused code

**Review by mtseluiko — APPROVED** [review]

**Pull request merged**